lib/facter/gluster.rb

```diff
@@ -61,28 +61,28 @@
   if peer_count > 0
     Facter.add(:gluster_peer_list) do
       setcode do
-        peer_list
+        peer_list.sort
       end
     end
 
     unless volume_bricks.empty?
       Facter.add(:gluster_volume_list) do
         setcode do
-          volume_bricks.keys
+          volume_bricks.keys.sort
         end
       end
       volume_bricks.each do |vol, bricks|
         Facter.add("gluster_volume_#{vol}_bricks".to_sym) do
           setcode do
-            bricks
+            bricks.sort
```

Comment:
Having read a bit more, I'm not actually sure this is the correct thing to do. Brick order *is* important?
See https://github.com/voxpupuli/puppet-gluster/issues/121#issuecomment-302011451